src/scalper.ts

```diff
@@ -667,32 +669,39 @@ export class Scalper {
       [`${this.symbol}/USDC`],
       (message: SerumVialTradeMessage) => {
         console.log(this.symbol, "Gamma Scalp Filled!", message.size, message.market, message.price, message.timestamp);
+        let fillPrice = Number(message.price);
         this.serumVialClient.removeAnyListeners();
         this.serumVialClient.closeSerumVial();
         this.gammaScalpOpenBook(
           dipProduct,
           gammaScalpCount + 1,
+          fillPrice
         );
       }
     );
 
     // Find fair value.
-    console.log(this.symbol, "Loading Fair Value...");
-    const pythPrice = await getPythPrice(new PublicKey(tokenToSplMint(this.symbol)));
-    const bids = await spotMarket.loadBids(this.connection);
-    const asks = await spotMarket.loadAsks(this.connection);
-    const [bidPrice, _bidSize] = bids.getL2(1)[0];
-    const [askPrice, _askSize] = asks.getL2(1)[0];
-    const midValue = (bidPrice + askPrice) / 2.0;
-    // Handle when pyth does not have a price
-    let fairValue
-    if (pythPrice > 0) {
-      fairValue = midValue*openBookLiquidityFactor + pythPrice*(1-openBookLiquidityFactor);
-      console.log(this.symbol, "Pyth Price", pythPrice, "OpenBook Mid Value", midValue, "Fair Value", fairValue);
-    } else {
-      fairValue = midValue;
-      console.log(this.symbol, "Pyth Price Bad Using OpenBook Mid Value", midValue, "Fair Value", fairValue);
-    }
+    console.log(this.symbol, "Loading Fair Value For Scalp", gammaScalpCount);
+    let fairValue;
+    if (gammaScalpCount==1){
```

Comment:
Should this be priorFillPrice == 0?
Functionally the same, but more accurate to use fill price since that is what it is really for.